Sources/SkeletonLayer.swift

```diff
@@ -53,6 +53,8 @@ struct SkeletonLayer {
         self.maskLayer.bounds = holder.maxBoundsEstimated
         addMultilinesIfNeeded()
         self.maskLayer.tint(withColors: colors)
+        
+        maskLayer.setValue(true, forUndefinedKey: "isSkeletonLayer")
```

Comment:
could we use `tag` instead?

Reply:
Hm, CALayer doesn't support tags (From what I know). Changed to name now instead as that seems to be more standard from what I have read about CALayers.